doc/news/changes/minor/20191016NiklasFehn

```diff
@@ -0,0 +1,5 @@
+BugFix: In GridGenerator::torus<3,3>() the radii set for the
+TorusManifold have been wrong, using some hard-coded values like 2.0
+and 0.5 instead of R and r.
+<br>
+(Niklas Fehn, 2019/10/16)
```

Comment:
This is correcting a change within the current 9.2-pre version of deal.II, namely #8197, for which we usually do not write changelog entry as this is not a change against a previous release version that needs to show up on [this page](https://www.dealii.org/developer/doxygen/deal.II/recent_changes.html). So I think we can drop this file here (which does not mean that your change is not appreciated).